modules/log6x-6-1-1-rn.adoc

```diff
@@ -0,0 +1,45 @@
+
+[id="logging-release-notes-6-1-1-enhancements_{context}"]
+== New Features and Enhancements
+
+* With this update, the {loki-op} supports configuring the workload identity federation on the {gcp-first} by using the Cluster Credential Operator (CCO) in {product-title} 4.17 or later. (link:https://issues.redhat.com/browse/LOG-6420[LOG-6420])
+
+[id="logging-release-notes-6-1-1-bug-fixes_{context}"]
+== Bug Fixes
+
+* Before this update, when any infrastructure namespaces were specified in the application inputs in the `ClusterLogForwarder` custom resource, logs were generated with the incorrect `log_type: application` tags. With this update, when any infrastructure namespaces are specified in the application inputs, logs are generated with the correct `log_type: infrastructure` tags. (link:https://issues.redhat.com/browse/LOG-6373[LOG-6373])
+
+* Before this update, the collector was discarding longer audit log messages with the following error message: *Internal log [Found line that exceeds max_line_bytes; discarding.]*. With this update, the discarding of longer audit messages is avoided by increasing the audit configuration thresholds: The maximum line size, `max_line_bytes`, is `3145728` bytes. The maximum number of bytes read during a read cycle, `max_read_bytes`, is `262144` bytes. (link:https://issues.redhat.com/browse/LOG-6379[LOG-6379])
+
+* Before this update, an input receiver service was repeatedly created and deleted, causing issues with mounting the TLS secrets. With this update, the service is created once and only deleted if it is not defined in the `ClusterLogForwarder` custom resource. (link:https://issues.redhat.com/browse/LOG-6383[LOG-6383])
+
+* Before this update, pipeline validation might have entered an infinite loop if a name was a substring of another name. With this update, stricter name equality checks prevent the infinite loop. (link:https://issues.redhat.com/browse/LOG-6405[LOG-6405])
+
+* Before this update, the collector alerting rules included the summary and message fields. With this update, the collector alerting rules include the summary and description fields. (link:https://issues.redhat.com/browse/LOG-6407[LOG-6407])
+
+* Before this update, setting up the custom audit inputs in the `ClusterLogForwarder` custom resource with configured `LokiStack` output caused errors due to the nil pointer dereference. With this update, the Operator performs the nil checks, preventing such errors. (link:https://issues.redhat.com/browse/LOG-6449[LOG-6449])
+
+* Before this update, the `ValidLokistackOTLPOutputs` condition appeared in the status of the `ClusterLogForwarder` custom resource even for non-`LokiStack` outputs. With this update, the `ValidLokistackOTLPOutputs` condition is removed, and the validation messages for the existing output conditions are corrected. (link:https://issues.redhat.com/browse/LOG-6469[LOG-6469])
```

Comment:
Not worth holding up the merge. But, seeing that the PR is on hold:

```suggestion
* Before this update, the `ValidLokistackOTLPOutputs` condition appeared in the status of the `ClusterLogForwarder` custom resource even when the output type is not LokiStack. With this update, the `ValidLokistackOTLPOutputs` condition is removed, and the validation messages for the existing output conditions are corrected. (link:https://issues.redhat.com/browse/LOG-6469[LOG-6469])
```

Reply:
TY. I applied the suggested change in the first sentence but left the second sentence as is because `are corrected` (=`have been corrected`) is the intended meaning.